queries.yaml

```diff
@@ -273,3 +273,28 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 9
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 14
+      properties: [is_bridge, is_tunnel, oneway]
+      where: >-
+        (kind == 'path' and zoom < 15) or
+        (kind in ['minor_road', 'major_road', 'highway', 'rail'] and zoom < 13)
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 16
+      properties: [name, ref, network]
+      where: >-
+        (kind == 'path' and zoom < 17) or
+        (kind in ['minor_road', 'rail'] and zoom < 15) or
+        (kind == 'major_road' and zoom < 12) or
+        (kind == 'highway' and zoom < 7)
```

Comment:
I'm all for more merging :)  Do we think others might have use of those
names though? There are cases like http://github.com/tangrams/explorer but
that one is pretty specific.

On Thu, Jan 7, 2016 at 11:02 AM, Nathaniel V. KELSO <
notifications@github.com> wrote:

> In queries.yaml
> <https://github.com/mapzen/vector-datasource/pull/468#discussion_r49088667>
> :
>
> > +      end_zoom: 14
> > +      properties: [is_bridge, is_tunnel, oneway]
> > +      where: >-
> > +        (kind == 'path' and zoom < 15) or
> > +        (kind in ['minor_road', 'major_road', 'highway', 'rail'] and zoom < 13)
> > +  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
> > +    params:
> > +      source_layer: roads
> > +      start_zoom: 0
> > +      end_zoom: 16
> > +      properties: [name, ref, network]
> > +      where: >-
> > +        (kind == 'path' and zoom < 17) or
> > +        (kind in ['minor_road', 'rail'] and zoom < 15) or
> > +        (kind == 'major_road' and zoom < 12) or
> > +        (kind == 'highway' and zoom < 7)
>
> This is good as-is, but optionally:
>
> Generally we don't label with name less than zoom 11, so you could also
> consider dropping name at zooms less than 11 from highways, that would
> probably result in a greater amount of line merging (a good thing).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mapzen/vector-datasource/pull/468/files#r49088667>.
>